pkg/deploy/deploy_test.go

```diff
@@ -1322,32 +1320,7 @@ func TestDeployConfigFF(t *testing.T) {
 		featureFlag       string
 		configType        config.TypeID
 		expectedErrString string
-	}{
-		{
-			name: "SLO FF test",
-			projects: []project.Project{
-				{
-					Configs: project.ConfigsPerTypePerEnvironments{
-						"env": project.ConfigsPerType{
-							"p1": {
-								config.Config{
-									Type:        config.ServiceLevelObjective{},
-									Environment: "env",
-									Coordinate: coordinate.Coordinate{
-										Project:  "p1",
-										Type:     "type",
-										ConfigId: "config1",
-									},
-								},
-							},
-						},
-					},
-				},
-			},
-			featureFlag: featureflags.ServiceLevelObjective.EnvName(),
-			configType:  config.ServiceLevelObjectiveID,
-		},
-	}
+	}{}
```

Comment:
Not sure about this one. At the moment, this one is empty. Should I completely remove it? We might need it again, when adding new configs